packages/core/addon/components/navi-visualization-config/table.hbs

```diff
@@ -11,23 +11,21 @@
       {{!-- Toggle for grand total --}}
       <div class="table-config__totals-toggle-section">
         <span class="table-config__totals-toggle-label">Grand Total</span>
-        <XToggle
+        <DenaliSwitch
           class="table-config__total-toggle-button table-config__total-toggle-button--grand-total"
-          @value={{readonly @options.showTotals.grandTotal}}
-          @size="small"
-          @onToggle={{this.onToggleGrandTotal}}
+          @isEnabled={{@options.showTotals.grandTotal}}
+          @onChange={{pipe (pick 'target.value') (fn this.onToggleGrandTotal )}}
```

Comment:
```suggestion
          @onChange={{pipe (pick "target.value") (fn this.onToggleGrandTotal )}}
```

Reply:
> Can we also remove `ember-toggle` from the package.json files since this should completely remove it

I believe that we still use it in other files? though they might need switched to this format because they're @onChange